package.json

```diff
@@ -0,0 +1,56 @@
+{
+  "bugs": {
+    "url": "https://github.com/adobe/aem-headless-client-nodejs/issues"
+  },
+  "bundleDependencies": false,
+  "dependencies": {
+    "@adobe/aem-headless-client-js": "git+https://git@github.com/adobe/aem-headless-client-js#feature/19-split-browser-and-nodejs",
```

Comment:
TODO: Should be updated after JS package is published

Reply:
We would use a released version or something like `^2.1.0`, not the `main` branch right?

Reply:
yes, once it's released

Reply:
I think that we need major version update for Browser version now, since `getToken` method is removed, now it's only part of NodeJS package. So I would go with 3.0.0 for Browser package